### PR TITLE
[RA2 Tech] Change containeri[s|z]ed to cloud native where appropriate

### LINKED ISSUE #1534: [RA-2] Remove containerised in RA-2
It is better to replace "containerised" with "cloud native" in [RA-2](https://github.com/cntt-n/CNTT/search?q=Containerised&type=Code)?

### PR DESCRIPTION
**Pull request by tomkivlin**
@YuPengZTE - I have not changed all instances as some are quotes or correct in context.  Hope it makes sense.
Fixes #1534

### CODE REVIEW

**Comment by tomkivlin** [comment]
@pgoyal01 not sure why the openstack page is listed - I didn't make any changes to it, and it looks like a bunch of end-of-line spaces have been removed, that's all.

File: doc/tech/glossary.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ A **Container** provides operating-system-level virtualisation by abstracting th
 - **Container Engine**: Software components used to create, destroy, and manage containers on top of an operating system.
 
 - **Pod**:	The smallest and simplest Kubernetes object. A Pod represents a set of running containers on your cluster. A Pod is typically set up to run a single primary container. It can also run optional sidecar containers that add supplementary features like logging.
-- **Kubernetes Cluster**: A set of machines, called nodes and master, that run containerised applications managed by Kubernetes. A cluster has at least one worker node and at least one master.
+- **Kubernetes Cluster**: A set of machines, called nodes and master, that run container-based and VM-based applications managed by Kubernetes. A cluster has at least one worker node and at least one master.
```
Comment (ulikleber) [inline doc/tech/glossary.md:99]: Kubernetes uses "containerized" in the cluster definition.

Comment (tomkivlin) [reply doc/tech/glossary.md:99]: Ah yes good spot - will reverse this.

Comment (tomkivlin) [reply doc/tech/glossary.md]: Corrected here: https://github.com/cntt-n/CNTT/pull/1652/commits/ef5b3eac5197a6429ee6a334e594123170c64776

File: doc/tech/glossary.md [hunk 2]
```diff
@@ -96,7 +96,7 @@ A **Container** provides operating-system-level virtualisation by abstracting th
 - **Container Engine**: Software components used to create, destroy, and manage containers on top of an operating system.
 
 - **Pod**:	The smallest and simplest Kubernetes object. A Pod represents a set of running containers on your cluster. A Pod is typically set up to run a single primary container. It can also run optional sidecar containers that add supplementary features like logging.
-- **Kubernetes Cluster**: A set of machines, called nodes and master, that run container-based and VM-based applications managed by Kubernetes. A cluster has at least one worker node and at least one master.
+- **Kubernetes Cluster**: A set of machines, called nodes and master, that run containerised applications managed by Kubernetes. A cluster has at least one worker node and at least one master.
```
Comment (CsatariGergely) [inline doc/tech/glossary.md]: Are we giving up on the dream to run VM-s also in Kubernetes with Kubevirt of Virtlet?

Comment (tomkivlin) [reply doc/tech/glossary.md]: No - but this is the definition from kubernetes.io, which we decided to pull definitions from.

File: doc/ref_arch/kubernetes/chapters/chapter05.md [hunk 3]
```diff
@@ -164,8 +164,8 @@ Sometimes container isolation is compared directly with VM based isolation, with
 - Containers: isolation by SW based mechanisms available in OS, Docker and Kubernetes. A container workload is just a set of Linux processes. It is _possible_ to configure SW based _additional isolation_ for container workloads, for example by kernel namespaces.
 
 
-Thus the primary isolation mechanism in Kubernetes environment should be VM or physical machine based isolation. This means: multiple container applications should not be deployed together in the same Kubernetes cluster - unless those have been planned and verified to co-exist. In such cases, the default is to allocate one namespace per Containerised Network Function (CNF).
+Thus the primary isolation mechanism in Kubernetes environment should be VM or physical machine based isolation. This means: multiple container applications should not be deployed together in the same Kubernetes cluster - unless those have been planned and verified to co-exist. In such cases, the default is to allocate one namespace per Cloud Native Network Function (CNF).
```
Comment (pgoyal01) [inline doc/ref_arch/kubernetes/chapters/chapter05.md:167]:
```suggestion
The primary isolation mechanism in Kubernetes environment should be VM or physical machine based. This implies that multiple cloud native applications should not be deployed together in the same Kubernetes cluster - unless these applications have been planned and verified to co-exist. Thus, the default is to allocate one namespace per Cloud Native Network Function (CNF).
```

File: doc/ref_arch/kubernetes/chapters/chapter05.md [hunk 4]
```diff
@@ -164,8 +164,8 @@ Sometimes container isolation is compared directly with VM based isolation, with
...
 
 ### Container Isolation in Kubernetes Cluster
 #### Namespaces  
-Kubernetes namespaces should be used to provide resource isolation within a Kubernetes cluster. They should not be used to isolate different steps in the deployment process like Development, Production, or Testing. The most reliable separation is achieved by deploying sensitive workloads into dedicated clusters. 
+Kubernetes namespaces should be used to provide resource isolation within a Kubernetes cluster. They should not be used to isolate different steps in the deployment process like Development, Production, or Testing. The most reliable separation is achieved by deploying sensitive workloads into dedicated clusters.
```
Comment (pgoyal01) [inline doc/ref_arch/kubernetes/chapters/chapter05.md:171]: Are "Development, Production, or Testing" steps in deployment process or they are different stages in the application development process.

File: doc/ref_arch/kubernetes/chapters/chapter05.md [hunk 5]
```diff
@@ -93,13 +93,13 @@ The following are recommended best practices for container run-time:
 - Integrate run-time processes to Security Information and Event Monitoring (SIEM)
 - Use container-aware run-time defense tools
 - Ensure all running container applications are from secure and verified images
```
Comment (pgoyal01) [inline doc/ref_arch/kubernetes/chapters/chapter05.md:95]:
```suggestion
- Ensure all running cloud native applications are from secure and verified images
```

File: doc/ref_arch/kubernetes/chapters/chapter05.md [hunk 6]
```diff
@@ -93,13 +93,13 @@ The following are recommended best practices for container run-time:
...
-- Containerised application should not run with root privileges
+- Cloud native application should not run with root privileges
```
Comment (pgoyal01) [inline doc/ref_arch/kubernetes/chapters/chapter05.md:96]:
```suggestion
- Cloud native applications should not run with root privileges
```

File: doc/ref_arch/kubernetes/chapters/chapter05.md [hunk 7]
```diff
@@ -93,13 +93,13 @@ The following are recommended best practices for container run-time:
...
 - Ensure sensitive workloads are properly segmented by namespaces or cluster to mitigate the scope of compromise.
 
 ##  5.12  Secrets Management
 The principle of least privilege must be applied to secret management in Kubernetes;
 
-- Ensure the containerised code can read only the secrets that it needs
+- Ensure the cloud native application code can read only the secrets that it needs
```
Comment (pgoyal01) [inline doc/ref_arch/kubernetes/chapters/chapter05.md:102]:
```suggestion
- Ensure that the cloud native applications can only read the secrets that these applications need
```

File: doc/ref_arch/kubernetes/chapters/chapter05.md (hunk 4 above)
Comment (tomkivlin) [reply doc/ref_arch/kubernetes/chapters/chapter05.md:171]: Not sure - note, I didn't actually make any changes here so would prefer to handle this in a separate issue/PR please.

**Pull request merged**